### ISSUE TITLE
colorize plugin not printing properly

### ISSUE THREAD

**Issue by jorgecoca** [jorgecoca/lumberdash#7]
![lumberdash](https://user-images.githubusercontent.com/3237451/54128469-24275600-43da-11e9-9bd8-e8596ae7fcd8.jpg)

**Comment by nodinosaur** [comment]
To expand upon this I have found that the colorize extension does not behave as expected... 

Import dependencies:
```
import 'package:lumberdash/lumberdash.dart';
import 'package:colorize_lumberdash/colorize_lumberdash.dart';
```

into any method add :
```
 putLumberdashToWork(withClient: ColorizeLumberdash());
 logWarning('Hello Warning');
 logFatal('Hello Fatal!');
 logMessage('Hello Message!');
 logError(Exception('Hello Error'));
```

Call the method

**Comment by jorgecoca** [comment]
Fixed on #8
